File: QolmsJotoWebView/QolmsJotoWebView/Models/InnerModels/PointExchangeHistItem.cs
using System;

namespace MGF.QOLMS.QolmsJotoWebView
{
    /// <summary>
    /// ポイント交換の履歴を表します。
    /// このクラスは継承できません。
    /// </summary>
    [Serializable()]
    public sealed class PointExchangeHistItem
    {
        #region Public Property

        /// <summary>
        /// 交換日時を取得または設定します。
        /// </summary>
        public DateTime IssueDate { get; set; } = DateTime.MinValue;

        /// <summary>
        /// 有効期限を取得または設定します。
        /// </summary>
        public DateTime ExpirationDate { get; set; } = DateTime.MinValue;

        /// <summary>
        /// クーポン種別を取得または設定します。
        /// </summary>
        public byte CouponType { get; set; } = byte.MinValue;

        /// <summary>
        /// クーポンコードを取得または設定します。
        /// </summary>
        public string CouponId { get; set; } = string.Empty;

        /// <summary>
        /// 表示名を取得または設定します。
        /// </summary>
        public string DispName { get; set; } = string.Empty;

        /// <summary>
        /// 消費ポイントを取得または設定します。
        /// </summary>
        public int Point { get; set; } = int.MinValue;

        #endregion

        #region Constructor

        /// <summary>
        /// <see cref="PointExchangeHistItem" /> クラスの新しいインスタンスを初期化します。
        /// </summary>
        public PointExchangeHistItem()
        {
        }

        #endregion
    }
}

File: QolmsYappli/QolmsYappli/Models/InnerModel/PointExchangeHistItem.vb
''' <summary>
''' ポイント交換の履歴を表します。
''' このクラスは継承できません。
''' </summary>
''' <remarks></remarks>
<Serializable()>
Public NotInheritable Class PointExchangeHistItem

#Region "Public Property"

    ''' <summary>
    ''' 交換日時を取得または設定します。
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Property IssueDate As Date = Date.MinValue

    ''' <summary>
    ''' 有効期限を取得または設定します。
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Property ExpirationDate As Date = Date.MinValue

    ''' <summary>
    ''' クーポン種別を取得または設定します。
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Property CouponType As Byte = Byte.MinValue

    ''' <summary>
    ''' クーポンコードを取得または設定します。
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Property CouponId As String = String.Empty

    ''' <summary>
    ''' 表示名を取得または設定します。
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Property DispName As String = String.Empty

    ''' <summary>
    ''' 消費ポイントを取得または設定します。
    ''' </summary>
    ''' <value></value>
    ''' <returns></returns>
    ''' <remarks></remarks>
    Public Property Point As Integer = Integer.MinValue

#End Region

#Region "Constructor"

    ''' <summary>
    ''' <see cref="PointExchangeHistItem" /> クラスの新しいインスタンスを初期化します。
    ''' </summary>
    ''' <remarks></remarks>
    Public Sub New()
    End Sub

#End Region

End Class
